thavalon-webapp/src/components/gameComponents/missionResults.tsx

```diff
@@ -1,5 +1,176 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { GameSocket, InboundMessage, InboundMessageType } from "../../utils/GameSocket";
+import { GameMessage, GameMessageType, MissionGoingMessage, MissionResultsMessage, Snapshot } from "./constants";
+import "../../styles/gameStyles/missionResults.scss";
+import { OverlayTrigger, Tooltip } from "react-bootstrap";
+
+enum MissionStatus {
+    Pending = "pending",
+    Going = "going",
+    Passed = "passed",
+    Failed = "failed"
+}
+
+interface MissionCardProps {
+    missionPlayers: String[],
+    missionStatus: MissionStatus,
+    missionNumber: number,
+    passes?: number,
+    fails?: number,
+    questing_beasts?: number,
+    reverses?: number,
+}
 
 export function MissionResults(): JSX.Element {
-    return <h1>Mission Results</h1>;
+    const [missionProps, setMissionProps] = useState<MissionCardProps[]>([
+        {
+            missionPlayers: ["Mission 1"],
+            missionStatus: MissionStatus.Pending,
+            missionNumber: 1,
+        },
+        {
+            missionPlayers: ["Mission 2"],
+            missionStatus: MissionStatus.Pending,
+            missionNumber: 2,
+        },
+        {
+            missionPlayers: ["Mission 3"],
+            missionStatus: MissionStatus.Pending,
+            missionNumber: 3,
+        },
+        {
+            missionPlayers: ["Mission 4"],
+            missionStatus: MissionStatus.Pending,
+            missionNumber: 4,
+        },
+        {
+            missionPlayers: ["Mission 5"],
+            missionStatus: MissionStatus.Pending,
+            missionNumber: 5,
```

Comment:
Could we use a for loop here instead of copying and pasting this five times, like I did with my ENGN homework that I was mocked for?